src/context.ts

```diff
@@ -0,0 +1,20 @@
+import * as vscode from 'vscode';
+import { SwiftPackage } from './package';
+
+export class SwiftContext {
+	private constructor(
+        public workspaceRoot: string,
+        public extContext: vscode.ExtensionContext,
```

Comment:
Last abbreviation nitpick 😅 Currently this property is a `SwiftContext.extContext: ExtensionContext`. Perhaps we can just name it `extensionContext`?

Reply:
ok

Reply:
done